app/assets/stylesheets/components/_typography.scss

```diff
@@ -14,6 +14,9 @@ body { -webkit-font-smoothing: antialiased; }
 .fs-12p { font-size: 12px; }
 .fs-20p { font-size: 20px; }
 
+.color-blue { color: $blue; }
+.color-navy { color: $navy; }
+
```

Comment:
@hursey013 I removed the redundant class definitions and also an unnecessary color class in the markup